ansible/roles/nginx-proxy/templates/nginx-confd-proxy.j2

```diff
@@ -1,17 +1,30 @@
 server {
     listen       80 default_server;
```

Comment:
maybe split them to two block rather then use if, see https://github.com/openmicroscopy/management_tools/blob/master/ci/config/cowfish/nginx/conf.d/OMERO-DEV-multiweb.conf#L13